packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettings.test.tsx

```diff
@@ -51,4 +51,28 @@ describe('DatabaseSettings Component', () => {
       expect(button).toBeEnabled();
     }
   });
+
+  it('Should render Maintenance Window with radio buttons', () => {
+    const database = databaseFactory.build({
+      platform: 'rdbms-legacy',
+    });
+    const { getByRole } = renderWithTheme(
+      <DatabaseSettings database={database} />
+    );
+    const radioInput = getByRole('radiogroup');
+    expect(radioInput).toHaveTextContent('Monthly');
+    expect(radioInput).toHaveTextContent('Weekly');
+    expect(screen.queryByText('Maintenance Window')).toBeTruthy();
+  });
+
+  it('Should render Weekly Maintenance Window', () => {
+    const database = databaseFactory.build({
+      platform: 'rdbms-default',
+    });
+    renderWithTheme(<DatabaseSettings database={database} />);
+
+    expect(screen.queryByText('Monthly')).toBeNull();
+    expect(screen.queryByText('Weekly')).toBeNull();
+    expect(screen.queryByText('Set a Weekly Maintenance Window')).toBeTruthy();
```

Comment:
nitpick: We could do something like this:
```suggestion
    const { queryByText } = renderWithTheme(<DatabaseSettings database={database} />);
    
    expect(queryByText('Monthly')).toBeNull();
    expect(queryByText('Weekly')).toBeNull();
    expect(queryByText('Set a Weekly Maintenance Window')).toBeTruthy();
```

so that we don't need to import screen

Reply:
Thanks, done